test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/Linux/AcceptanceTest.cs

```diff
@@ -155,12 +155,18 @@ public Task ResourceUtilizationTracker_Reports_The_Same_Values_As_One_Can_Observ
         var memoryFromGauge = 0.0d;
         using var e = new ManualResetEventSlim();
 
-        listener.InstrumentPublished = (i, m) =>
+        object? meterScope = null;
+        listener.InstrumentPublished = (instrument, meterListener) =>
         {
-            if (i.Name == ResourceUtilizationInstruments.CpuUtilization
-            || i.Name == ResourceUtilizationInstruments.MemoryUtilization)
+            if (!ReferenceEquals(instrument.Meter.Scope, meterScope))
```

Comment:
This fixes a flaky test: https://dev.azure.com/dnceng-public/public/_build/results?buildId=458782&view=ms.vss-test-web.build-test-results-tab&runId=10343638&resultId=100183&paneView=debug